dnf-plugins/py3query.py

```diff
@@ -268,7 +269,14 @@ def bug_namegetter(bug):
                 status = bug.status
                 if bug.resolution:
                     status += ' ' + bug.resolution
-                r.setdefault('links', {})['bug'] = [bug.weburl, status]
+                # Let's get the datetime of the last comment
+                last_comment_datetime = None
+                if bug.comments:
```

Comment:
Yes, `last_change_time` might be even better than last commit. It's definitely not worse, IMO.